### PR TITLE
Volumes doc

### PR DESCRIPTION
**Pull request by mattmanning**
Doc explaining volume mounting. Should be released at the same time as Rack EFS integration.

I tried to keep this short and sweet like the other docs and show a clear example, but open to suggestions for more.

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;HcqWlvrI&#x2F;310-doc-for-shared-volumes"> doc for shared volumes</a>

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;HcqWlvrI&#x2F;310-doc-for-shared-volumes"> doc for shared volumes</a>

File: _deployment/volumes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+[WordPress](https://wordpress.com/) is a popular PHP blogging platform. It expects a persistent filesystem for storing themes, plugins, and media uploads. Rack Processes are stateless and disposable by default, but you can persist files using volumes as shown in the following example docker-compose.yml`.
+
+```
+web:
+  image: wordpress:4.5.2-apache
+  ports:
+    - 80:80
+  volumes:
+    - /var/www/html
```
Comment (ddollar) [inline _deployment/volumes.md]: Should we share the whole thing or just the wp-config directory?

**Comment by mwarkentin** [comment]
I wonder if Jenkins would be another interesting example, as another heavily file-system based project.

File: _deployment/volumes.md (hunk 1 above)
Comment (mattmanning) [inline _deployment/volumes.md]: The whole thing. There was some weird behavior in Wordpress updates (can't remember the details) when sharing just wp-content.

**Comment by mwarkentin** [comment]
Have you seen any good info out there about backups for EFS volumes? Can we just trust that AWS will never lose any data on EFS, or should we be looking at manual backups stored elsewhere?

**Pull request merged**